kitsune/search/tasks.py

```diff
@@ -121,7 +121,7 @@ def index_chunk_task(write_index, batch_id, chunk):
 
     except Exception:
         rec.text = u'{0}: Errored out {1} {2}'.format(
-            rec.text, sys.exc_type, sys.exc_value)
+            rec.text, sys.exc_type, sys.exc_value)[:255]  # Truncate at 255 chars.
```

Comment:
I don't know how big of an error it was. I was going to try this first and see what it looks like. Then increase the column size if that was needed.